config/environments/development.rb

```diff
@@ -59,5 +59,7 @@
       environment variable.
     MESSAGE
   end
+
+  config.hosts << "publishing-api.dev.gov.uk"
```

Comment:
The link "Issue on govuk-docker" in this commit links to a PR in the 'support' repo. Could you fix it?

Reply:
To me this reads more like a prior art example, so maybe the description of that link could do with changing but I think it's still the right thing to link to. In fact following through from that commit I think you were the first person to encounter this back in august with link-checker-api?

Reply:
Yes I think I was! Updating the link description would be fine.